src/localisation/localisation.h

```diff
@@ -88,4 +88,7 @@ static inline void set_format_arg_body(uint8 *args, size_t offset, uintptr_t val
 #define set_map_tooltip_format_arg(offset, type, value) \
 	set_format_arg_body(gMapTooltipFormatArgs, offset, (uintptr_t)value, sizeof(type))
 
+money32 string_to_money(char * string_to_monetise);
```

Comment:
I couldn't figure out exactly where would be an appropriate placement in the file, plus my medicine began to ware off so i began to struggle concentrating. where do you think would be a good placement?